src/UXClient/Utils.ts

```diff
@@ -867,20 +867,18 @@ class Utils {
             let dataGroupKeyedObject = {};
             dataGroupKeyedObject[dGName] = convertedDataGroup;
             convertedData.push(dataGroupKeyedObject);
-            if (dataGroup) {
-                Object.keys(dataGroup).forEach((seriesName) => {
-                    convertedDataGroup[seriesName] = {};
-                    Object.keys(dataGroup[seriesName]).forEach((rawTS: string) => {
-                        let isoString: string;
-                        try {
-                            isoString = (new Date(rawTS)).toISOString();
-                            convertedDataGroup[seriesName][isoString] = dataGroup[seriesName][rawTS];
-                        } catch(RangeError) {
-                            console.log(`${rawTS} is not a valid ISO time`);
-                        }
-                    });
+            dataGroup && Object.keys(dataGroup).forEach((seriesName) => {
```

Comment:
i think we use conventional if() blocks here instead of &&, just a slightly more traditional code flow instead of using boolean evaluation to side-effect.  i get that we do it all the time in tsx, but that also doesnt allow if() statements so it makes sense there